From 5c6c0345ef83edf6d9752e5574b83c50457011bf Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Wed, 5 Aug 2020 19:30:00 +0200
Subject: [PATCH] IWYU: std::vector used in fuzzy_finder.h requires including
 <vector>
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Build with libstdc++ fails with:
In file included from ../../chrome/browser/ui/commander/fuzzy_finder.cc:5:
../../chrome/browser/ui/commander/fuzzy_finder.h:30:23: error: ‘std::vector’ has not been declared
   30 |                  std::vector<gfx::Range>* matched_ranges);
      |                       ^~~~~~

Bug: 957519
Change-Id: Id290b8f93e0f5741d88d6a6e4c75f3c0c0e9f636
---
 chrome/browser/ui/commander/fuzzy_finder.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/chrome/browser/ui/commander/fuzzy_finder.h b/chrome/browser/ui/commander/fuzzy_finder.h
index 708b981b2004..4b017970b590 100644
--- a/chrome/browser/ui/commander/fuzzy_finder.h
+++ b/chrome/browser/ui/commander/fuzzy_finder.h
@@ -5,6 +5,8 @@
 #ifndef CHROME_BROWSER_UI_COMMANDER_FUZZY_FINDER_H_
 #define CHROME_BROWSER_UI_COMMANDER_FUZZY_FINDER_H_
 
+#include <vector>
+
 #include "base/strings/string16.h"
 #include "ui/gfx/range/range.h"
 
-- 
2.25.1

